ResearchKit/Common/ORKHelpers.m

```diff
@@ -545,3 +545,12 @@ void ORKAdjustPageViewControllerNavigationDirectionForRTL(UIPageViewControllerNa
         *direction = (*direction == UIPageViewControllerNavigationDirectionForward) ? UIPageViewControllerNavigationDirectionReverse : UIPageViewControllerNavigationDirectionForward;
     }
 }
+
+NSString *ORKPaddingWithNumberOfSpaces(NSUInteger numberOfPaddingSpaces) {
+    NSMutableString *padding = [NSMutableString new];
+    while (numberOfPaddingSpaces) {
```

Comment:
Can you try this API: 
https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSString_Class/#//apple_ref/occ/instm/NSString/stringByPaddingToLength:withString:startingAtIndex: